qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/CreateCustomerView.groovy

```diff
@@ -91,20 +97,29 @@ class CreateCustomerView extends VerticalLayout {
         row2.setDefaultComponentAlignment(Alignment.BOTTOM_LEFT)
         row2.setSizeFull()
 
-        HorizontalLayout row3 = new HorizontalLayout(affiliationComboBox, affiliationButton, submitButton)
+        HorizontalLayout row3 = new HorizontalLayout(affiliationComboBox, addressAdditionComboBox)
```

Comment:
NIT: can these be named in a more meaningful way? Maybe:
titleRowLayout, personalInfoLayout, affiliationLayout etc.

Reply:
I did not do that since there is still change expected and the position on screen does not necessarily mean that all elements in one row share a context e.g. the submit button would be beside the first name in row1 and the email together with the gender in row2 🤷 

Therefore I kept it descriptive as to the position it will occur in the layout.

Reply:
okay, that reason is good enough for me 🤷‍♂️